markdownlint/docs/TOP003.md

```diff
@@ -0,0 +1,143 @@
+# `TOP003` - Default section content
+
+Tags: `content`
+
+Aliases: `default-section-content`
+
+This rule is triggered when a section does not have the expected default content or structure. The affected sections include:
+
+- Lesson Overview
+- Assignment
+- Knowledge Check
+- Additional Resources
+
+## Contains content before or after list
+
+There should be no content before or after the unordered list of items in the "Lesson overview", "Knowledge check", and "Additional resources" sections, other than the default content.
+
+Incorrect (content after the list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+
+Some additional content
+```
+
+Incorrect (content before the list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+Some additional content
+
+- LO item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+## Contains incorrect default content
+
+Each section should contain the correct default content.
+
+Incorrect (wrong default content before the list):
+
+```markdown
+### Lesson Overview
+
+This section has the incorrect text following the heading.
+
+- LO item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+Incorrect (missing the list after the `Additional Resources` heading):
+
+```markdown
+### Additional Resources
+
+This section contains helpful links to related content. It is not required, so consider it supplemental.
+```
+
+Correct:
+
+```markdown
+### Additional Resources
+
+This section contains helpful links to related content. It is not required, so consider it supplemental.
+
+- AR item
+```
+
+Incorrect (contains a nested list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- An item
+  - A nested item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- An item
+- Another item
+```
+
+## Missing "Assignment" section div wrapper
+
+The "Assignment" section ordered list should be wrapped in a div with the correct attributes.
```

Comment:
```suggestion
The "Assignment" section should contain a div with the correct attributes. Note that not all assignment content needs to be wrapped in this div, but it must at least exist in this section.
```

Reply:
Not all assignment content?

Reply:
Not currently no. Maybe something to revisit, but right now we're just checking whether at least one of these div wrappers exists in the Assignment section.